src/Core/DefaultGeometryLayers.js

```diff
@@ -124,7 +138,21 @@ export function createGlobe(id, options) {
     return wgs84TileLayer;
 }
 
-
+/**
+ * Creates a planar geometry, which can be used to draw other layers (color, elevation, ...)
```

Comment:
"Creates a planar geometry, **on** which other layers (color, elevation, ...) **can be drawn**"